node/pkg/processor/processor.go

```diff
@@ -23,7 +26,7 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promauto"
-	dto "github.com/prometheus/client_model/go"
+	// dto "github.com/prometheus/client_model/go"
```

Comment:
Remove?

Reply:
No, I want to keep that because it goes with the commented out code to dump the metrics on shutdown, which I want to keep because it's useful when running benchmarks.